src/main/java/org/snomed/aag/data/services/AcceptanceService.java

```diff
@@ -156,7 +153,7 @@ public void processCommit(CommitInformation commitInformation) {
 	public void processValidationAsync(ValidationInformation validationInformation) {
 		try {
 			final String branchPath = validationInformation.getBranchPath();
-			final ProjectAcceptanceCriteria criteria = criteriaService.findEffectiveCriteriaWithMandatoryItems(branchPath);
+			final ProjectAcceptanceCriteria criteria = criteriaService.findByBranchPathWithRelevantCriteriaItems(branchPath);
```

Comment:
I renamed the method from findEffectiveCriteriaWithMandatoryItems to findByBranchPathWithRelevantCriteriaItems as it felt more descriptive.